task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -8,6 +8,8 @@ metadata:
     tekton.dev/tags: "appstudio, hacbs"
   name: fbc-validation
 spec:
+  description: >-
+    Validate the fbc image
```

Comment:
```suggestion
    Validate the file-based-catalog (fbc) image
```

Reply:
SR: Validates the file-based catalog (FBC) image.

Reply:
Updated.